packages/library/.stylelintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    rules: {},
+    ignoreFiles: ['packages/library/**/main*.css'],
```

Comment:
add stylelint config to ignore linting in `main.css`, which is generated in build time by `tailwindcss` build